training/general/src/environments/pytorch-1.13-cuda11.7/context/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1130:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.20.2' \
+                'azureml-core=={{latest-pypi-version}}' \
+				'azureml-dataset-runtime=={{latest-pypi-version}}' \
+                'azureml-defaults=={{latest-pypi-version}}' \
+				'azure-ml=={{latest-pypi-version}}' \
+				'azure-ml-component=={{latest-pypi-version}}' \
+                'azureml-mlflow=={{latest-pypi-version}}' \
+		        'azureml-contrib-services=={{latest-pypi-version}}' \
+		        'azureml-contrib-services=={{latest-pypi-version}}' \
+                'torch-tb-profiler~=0.4.0' \
+				'py-spy==0.3.12' \
+		        'debugpy~=1.6.3'
+RUN pip install \
+	azure-ai-ml==1.2.0 \
+        azureml-inference-server-http~=0.7.0 \
+        inference-schema~=1.4.2.1 \
+        MarkupSafe==2.0.1 \
+	    regex \
+	    pybind11
+# Inference requirements
+COPY --from=mcr.microsoft.com/azureml/o16n-base/python-assets:20220607.v1 /artifacts /var/
+RUN /var/requirements/install_system_requirements.sh && \
+    cp /var/configuration/rsyslog.conf /etc/rsyslog.conf && \
+    cp /var/configuration/nginx.conf /etc/nginx/sites-available/app && \
+    ln -sf /etc/nginx/sites-available/app /etc/nginx/sites-enabled/app && \
+    rm -f /etc/nginx/sites-enabled/default
+ENV SVDIR=/var/runit
+ENV WORKER_TIMEOUT=400
+EXPOSE 5001 8883 8888
+# support Deepspeed launcher requirement of passwordless ssh login
+RUN apt-get update
+RUN apt-get install -y openssh-server openssh-client
```

Comment:
Please add some newlines here for readability. Also, is that the latest version of the `python-assets` image?

Reply:
Not sure, but this environment got recently added and this is the image it is using